_posts/2022-05-17-release-1.0.md

```diff
@@ -6,13 +6,13 @@ author: Rosemary Orchard
 ---
 Today, we are proud and excited to launch FocusCuts. 
 
-When WWDC happened last year we got some great things, namely Shortcuts for the Mac and Focus Modes. But, there was one omission–at least in our view. Shortcuts can't tell you if a Focus Mode is enabled or, which Focus Mode is active. We quickly realised this information isn't something you can get on iOS, but you can find it on the Mac!
+At WWDC 2021, the Mac received some incredible things, especially with the addition of Shortcuts for the Mac and Focus Modes. These two features further cemented the Mac as an incredible tool for power users, power, there was one omission—at least in our view. With Shortcuts, there is no way to tell if a Focus Mode is enabled or which Focus Mode is active. We quickly realised this information isn't something you can get on iOS, but you can find it on the Mac!
```

Comment:
There were a couple of typos in my suggestion.  And then I basically ended up rewriting the entire thing…

```suggestion
At WWDC 2021, Apple announced that the Mac would be receiving some incredible updates with the release of macOS Monterey.  These announcements, especially the addition of Shortcuts for the Mac and Focus Modes further cemented the Mac as an excellent tool for power users.  In our view, however, there was one omission. There is no way to tell if a Focus Mode is enabled or which Focus Mode is active with Shortcuts. We quickly realised this information isn't something you can find on iOS, but you can on the Mac!
```